### ISSUE TITLE
Number string not as string when checking

### ISSUE THREAD

**Issue by tin-pham** [AyubTouba/nestjs-dbvalidator#42]
Great library so far, save me tons of time, anything go to service is more logic beside just pure checking for uniqueness and existence. But i just got this problem.

Basically, I just check phone uniqueness, I send "222222222" , it not throw "22222222" is not number string, so that obviously a number string, but as the log show below, it try to check for 222222222 not "222222222" 
```
  @isUniqueDb({
    table: DATABASE_TABLE.USERS,
    column: USER_SCHEMA.PHONE,
  })
  @IsNumberString()
  @IsNotEmpty({
    message: 'Số điện thoại không được trống',
  })
  phone: string;

```

```
  query: 'select "phone" from "users" where  "phone" = 22222222   ',
  parameters: undefined,
  driverError: error: operator does not exist: character varying = integer

```

I also copy the log to pgadmin sql, it also the same error
![image](https://github.com/AyubTouba/nestjs-dbvalidator/assets/20679090/b50174e8-eb29-4f02-901a-21e48931dca6)

**Comment by AyubTouba** [comment]
First I'm glad that the library is helping you, 
for your question, the library's behavior is considered standard because it checks whether the value is a number or a string. If it's a number, it processes it without the quotes. However, I plan to introduce a parameter that allows you to select how you want to handle the value.

**Comment by tin-pham** [comment]
> First I'm glad that the library is helping you, for your question, the library's behavior is considered standard because it checks whether the value is a number or a string. If it's a number, it processes it without the quotes. However, I plan to introduce a parameter that allows you to select how you want to handle the value.

I'm looking forward to it

**Comment by tin-pham** [comment]
> First I'm glad that the library is helping you, for your question, the library's behavior is considered standard because it checks whether the value is a number or a string. If it's a number, it processes it without the quotes. However, I plan to introduce a parameter that allows you to select how you want to handle the value.

Recently i try to convert my table id column from varchar type to uuid type, it error out, i think it have to do with column type

**Comment by AyubTouba** [comment]
Hello @tin-pham,
I've just published Version 2.0.0 of the library, which is now compatible with Nest V9 and above. Additionally, I've implemented your requested feature. You can now choose the type of validation by adding the `customType` property and assigning it a type from the `TYPECOLUMN` enum. Feel free to explore the new functionalities!
